model-engine/model_engine_server/inference/batch_inference/dto.py

```diff
@@ -0,0 +1,165 @@
+# This is a copy of model_engine_server.common.dtos.llm
```

Comment:
should we have a pointer from model_engine_server.common.dtos.llm to here as well? so it's easier to remember to keep the two in sync